source/data_api.html.md

```diff
@@ -135,24 +137,44 @@ CURL example not available
 
 Welkin's APIs are secured using a 2-legged OAuth JWT-Bearer authorization flow. This ensures that data stored within Welkin remains secure and is not accessible by unauthorized parties.
 
-Once you obtain an access token, the token can be passed as an Authorization header along with the keyword `Bearer`.
+#### Authentication Steps:
+
+1. Get access to [Integration Tools](https://workshop.welkinhealth.com/integration-tools) (requires a Welkin Workshop login).
+2. Create a Client and an associated Credential.
+3. Select the scopes that the Client can request.
+4. Exchange Client Credential for an Access Token as shown in the example code.
+5. Make API requests with the Access Token.
+
+Once you obtain an Access Token, the token can be passed as an Authorization header along with the keyword `Bearer`.
+
+The Access Tokens expire after 1 hour and a new Access Token must be requested.
 
 More information on the JWT protocol can be found at [jwt.io](https://jwt.io/).
 
 **Token endpoint**: `https://api.welkinhealth.com/v1/token`
 
 **Expected JWT fields**
 
-* `iss` = your client_id as issued by Welkin
+* `iss` = Your client_id as issued by Welkin in Integration Tools
 * `aud` = `https://api.welkinhealth.com/v1/token`
 * `exp` = ISO timestamp when the token should expire (recommended 1 hours from current time)
-* `scope` = `all`
+* `scope` = A space separated string of scopes
 
 **Expected POST body fields**
 
 * `assertion` = JWT token
 * `grant_type` = `urn:ietf:params:oauth:grant-type:jwt-bearer`
 
+#### Scopes
+
+Scopes limit the types of data and actions that Client can take via the API. Scopes are passed as a space separated list when requesting an Access Token. For example: `calls.read patients.write assessments.read`
+
+Each resources has a `read` and a `write` scope.
+
+If you make a `POST` or `PUT` request with an access token which has only `write` scopes, you will receive a response with only the ID of the created or modified record. The rest of the fields will be redacted since you don't have `read` scope.
```

Comment:
Is it worth adding information about why this is the case? ie. to avoid leaking PHI or no?

Reply:
i don't think so